src/bsp/bm_mote_rs232/Drivers/STM32U5xx_HAL_Driver/Src/stm32u5xx_ll_usart.c

```diff
@@ -0,0 +1,446 @@
+/**

```

Comment:
Same as the .h, Until we make serial.c agnostic to USART v.s. LPUART, we need to make sure this stays in the bsp.